editor/src/core/shared/math-utils.ts

```diff
@@ -291,6 +291,19 @@ export function product<C extends CoordinateMarker>(a: Point<C>, b: Point<C>): n
   return a.x * b.x + a.y * b.y
 }
 
+export function pointIsClockwiseFromLine<C extends CoordinateMarker>(
+  targetPoint: Point<C>,
+  linePointA: Point<C>,
+  linePointB: Point<C>,
+): boolean {
+  // The order of line points a and b are important, as they determine which direction the line is pointing in,
```

Comment:
if the line is pointing towards a direction, shouldn't we call it a vector instead?